addons/tests/iss_health_check/test_iss_health_cheker.py

```diff
@@ -0,0 +1,142 @@
+from unittest.mock import patch, MagicMock, Mock

```

Comment:
It looks like this filename has a typo, should it be called `test_iss_health_checker.py`?